server/paymentProviders/paypal/payment.js

```diff
@@ -88,10 +102,10 @@ export async function executePayment(order) {
 export async function createTransaction(order, paymentInfo) {
   /* The `* 100` in the next lines convert from PayPal format in
      dollars to Open Collective format in cents */
-  const amountFromPayPal = paymentInfo.transactions.map(t => parseFloat(t.amount.total)) * 100;
-  const paypalFee =
-    paymentInfo.transactions.map(t => t.related_resources.map(r => parseFloat(r.sale.transaction_fee.value))) * 100;
-  const currencyFromPayPal = paymentInfo.transactions.map(t => t.amount.currency)[0];
+  const transaction = paymentInfo.transactions[0];
+  const amountFromPayPal = parseFloat(transaction.amount.total) * 100;
+  const paypalFee = parseFloat(get(transaction, 'related_resources.0.sale.transaction_fee.value', '0.0')) * 100;
```

Comment:
`related_resources.0.sale` can be null if there's no PayPal fee. This can happen in dev environment, this change will make it work.